modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/varbinary/ItVarBinaryDmlTest.java

```diff
@@ -70,4 +71,10 @@ public void testInsertHexLiteral() {
     protected DataTypeTestSpec<VarBinary> getTypeSpec() {
         return DataTypeTestSpecs.VARBINARY_TYPE;
     }
+
+    @Override
+    public void testUpdateFromLiteral(TestTypeArguments<VarBinary> arguments) {
```

Comment:
seems you can remove this test from base class too

Reply:
yep, thanks. Foxed